jenkins/helper/testing_runner.py

```diff
@@ -446,9 +448,15 @@ def generate_test_report(self):
             for subsubdir in subdir.iterdir():
                 path_segment = subsubdir.parts[len(subsubdir.parts) - 1]
                 if path_segment.startswith('arangosh_'):
+                    clean_subdir = True
                     for subsubsubdir in subsubdir.iterdir():
-                        shutil.move(str(subsubsubdir), str(subdir))
-                    subsubdir.rmdir()
+                        try:
+                            shutil.move(str(subsubsubdir), str(subdir))
+                        except shutil.Error as ex:
+                            print(f"failed to move file while cleaning up temporary files {ex}")
+                            clean_subdir = False
```

Comment:
Same here, this shouldn't happen. I wouldn't ignore errors unless we know why they happen and it's actually safe to ignore them.

Reply:
but if it does, we want the report anyways!

Reply:
I see your point. But on the other hand, packing the _wrong_ result from another run and reporting that instead could be even more annoying. Because then our report might not be empty, but can be totally misleading. I'd rather have an empty report than the wrong one.

Reply:
I don't get how having a report, but from another run, is better than having no report.

Reply:
Can we exclude the directories that we failed to move?

Reply:
Reporting it like you added now seems sensible. Can you add it at the other catch as well, and also make the message somewhat more helpful?
Like adding the affected path (unless it's part of Python's error message anyway), and a hint what that means for the report (i.e. that we might have packed something in the artifacts that doesn't belong to the run).